engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetCameraToReflected.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.stateChanges;
+
+import java.util.Objects;
+import org.terasology.rendering.cameras.Camera;
+import org.terasology.rendering.dag.RenderPipelineTask;
+import org.terasology.rendering.dag.StateChange;
+import org.terasology.rendering.dag.tasks.SetCameraReflectedModeTask;
+
+/**
+ * Instances of this state change generate the tasks that set or reset the reflected flag of a given camera.
+ */
+public class SetCameraToReflected implements StateChange {
```

Comment:
Just for a brief second - but the inclusion of "Camera" makes it pretty clear cut I think :-)

Any additional confirmation of details would be in the Javadoc which is close by hand anyway.